### PR TITLE
Finance status update redemption

### PR DESCRIPTION
**Pull request by puritybirir**
#### What does this PR do?
- Finance member should be able to log in 
- Once logged in, finance member should be able to see the approved request
- Finance member should be able to complete the request and also see the center of the redemption

#### Description of Task to be completed?

- As a user logged in as a member of the finance department, when a redemption request is approved I should get an email notification with a link to view more details about the redemption. Notifications module

#### Any background context you want to provide?

- The notification module is not yet done from the backend and there will therefore be a different bug for the same

- Also to be **Noted**, I added a pull request template for consistency

#### Questions:

- N/A

### CODE REVIEW

File: src/components/activities/ActivityCard.jsx [hunk 1]
```diff
@@ -96,7 +98,7 @@ class ActivityCard extends Component {
     super(props);
     this.state = {
       isActivityChecked: false,
-      statuses: ['pending', 'rejected', 'approved', 'in review'],
+      statuses: ['pending', 'rejected', 'approved', 'in review', 'completed'],
```
Comment (kimani-ndegwa) [inline src/components/activities/ActivityCard.jsx]: What do you think about having this as a map somewhere outside this component that we can reuse when necessary?

Also, what are `needButtons`?

File: src/components/activities/ActivityCard.jsx [hunk 2]
```diff
@@ -193,22 +199,39 @@ class ActivityCard extends Component {
...
+          className='verifyButtons__button verifyButtons__button--moreInfo'
+          onClick={() => handleClick(COMPLETE, id)}
         />
-        { moreInfoButtonHtml }
-      </div>
+      );
+    }
+    return (
+      <Fragment>
+        {
```
Comment (kimani-ndegwa) [inline src/components/activities/ActivityCard.jsx]: Why not have this as the same logic for `{moreInfoButtonHtml}` where you evaluate the result outside and then call the variable here?

File: src/components/activities/ActivityCard.jsx [hunk 3]
```diff
@@ -147,7 +154,9 @@ class ActivityCard extends Component {
       );
     }
 
-    let statusText = status.charAt(0).toUpperCase();
+    let statusText = status
```
Comment (mkiterian) [inline src/components/activities/ActivityCard.jsx]: We actually have a helper for this `capitalizeString`

**Comment by kimani-ndegwa** [comment]
Hey @puritybirir. Please rebase this.

**Pull request merged**